spec.md

```diff
@@ -377,38 +385,24 @@ both appear within the same JSON object.
   `contentType` values are defined in the event format specifications; for
   example, the JSON event format defines the relationship in
   [section 3.1](./json-format.md#31-special-handling-of-the-data-attribute).
-  
-  When this attribute is omitted, the "data" attribute simply follows the 
-  event format's encoding rules. For the JSON event format, the "data" 
-  attribute value can therefore be a JSON object, array, or value.   
-  
+
+  When this attribute is omitted, the "data" attribute simply follows the
```

Comment:
Nit: It's often best to avoid the word "simply" in documentation.

Reply:
I was *simply* copying the old text LOL
But I removed it